{"code":"EFATAL","level":"error","message":"Error setting webhook: EFATAL: AggregateError","service":"telegram-bot","stack":"RequestError: AggregateError\n    at new RequestError (C:\\Users\\DEll\\Desktop\\cursor\\number bot32\\number bot\\backend\\node_modules\\request-promise-core\\lib\\errors.js:14:15)\n    at plumbing.callback (C:\\Users\\DEll\\Desktop\\cursor\\number bot32\\number bot\\backend\\node_modules\\request-promise-core\\lib\\plumbing.js:87:29)\n    at Request.RP$callback [as _callback] (C:\\Users\\DEll\\Desktop\\cursor\\number bot32\\number bot\\backend\\node_modules\\request-promise-core\\lib\\plumbing.js:46:31)\n    at self.callback (C:\\Users\\DEll\\Desktop\\cursor\\number bot32\\number bot\\backend\\node_modules\\@cypress\\request\\request.js:183:22)\n    at Request.emit (node:events:507:28)\n    at Request.onRequestError (C:\\Users\\DEll\\Desktop\\cursor\\number bot32\\number bot\\backend\\node_modules\\@cypress\\request\\request.js:869:8)\n    at ClientRequest.emit (node:events:507:28)\n    at emitErrorEvent (node:_http_client:104:11)\n    at TLSSocket.socketErrorListener (node:_http_client:518:5)\n    at TLSSocket.emit (node:events:507:28)\n    at emitErrorNT (node:internal/streams/destroy:170:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:129:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-02-14 06:38:31"}
